integration-cli/docker_cli_run_test.go

```diff
@@ -477,6 +477,42 @@ func TestRunWithVolumesFromExited(t *testing.T) {
 	logDone("run - regression test for #4979 - volumes-from on exited container")
 }
 
+// Test create volume in a dirctory which is a symbolic link
+func TestRunCreateVolumesInSymlinkDir(t *testing.T) {
+	defer deleteAllContainers()
+	// This test has to create a file on host
+	hostFile := "/tmp/abcd"
```

Comment:
@cpuguy83 In this test, we create(by run and commit) an image which rootfs has a  `/home/test`  symlink to `/tmp`,  we use this image to run a container and create a volume `/home/test/abcd`,  
`parseVolumeMountConfig` will link `/home/test/abcd` to the `/tmp/abcd` of the host if no `symlink.FollowSymlinkInScope`, and the error will happen,. So we have to create a file on the
host to make sure it will not link to the host's rootfs